src/cc/libbpf.c

```diff
@@ -195,6 +195,7 @@ int bpf_prog_load(enum bpf_prog_type prog_type,
     // so repeat the syscall and print the result to stderr
     for (;;) {
          bpf_log_buffer = malloc(buffer_size);
+         memset(bpf_log_buffer, 0, buffer_size);
```

Comment:
lol. thanks for catching this bug :)
samples/bpf/libbpf.c is doing bpf_log_buf[0] = 0;
probably makes sense to do the same here.